compiler-plugin-tests/src/test/java/io/ballerina/lib/cdc/compiler/CompilerPluginTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2025, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Check for other places as well.
```suggestion
 * Copyright (c) 2025, WSO2 Inc. (http://www.wso2.org).
```